Source/Core/VideoCommon/VertexLoaderBase.cpp

```diff
@@ -96,45 +96,32 @@ class VertexLoaderTester : public VertexLoaderBase
   std::vector<u8> buffer_b;
 };
 
-template <class Function>
-static void GetComponentSizes(const TVtxDesc& vtx_desc, const VAT& vtx_attr, Function f)
+u32 VertexLoaderBase::GetVertexSize(const TVtxDesc& vtx_desc, const VAT& vtx_attr)
 {
-  if (vtx_desc.low.PosMatIdx)
-    f(1);
-  for (auto texmtxidx : vtx_desc.low.TexMatIdx)
-  {
-    if (texmtxidx)
-      f(1);
-  }
+  u32 size = 0;
+
+  size += __builtin_popcount(vtx_desc.low.Hex & 0x1FF);
```

Comment:
`Common/BitSet.h` has a Common::CountSetBits function that is supported on all platforms.

Reply:
Done.

Reply:
It looks like you added an include for `Common/BitSet.h`, but didn't actually switch to using `Common::CountSetBits` here?